graphs/bellman_ford.py

```diff
@@ -1,56 +1,40 @@
-from typing import Dict, List
-
-
-def printDist(dist, V):
-    print("Vertex Distance")
```

Comment:
The original has 4 space formatting and is compliant with psf/black as discussed in CONTRIBUTING.md.